libs/libcups/Makefile

```diff
@@ -0,0 +1,97 @@
+#
+# Copyright (C) 2006-2012 OpenWrt.org
+# Copyright (C) 2017 Luiz Angelo Daros de Luca <luizluca@gmail.com>
+#
+# This is free software, licensed under the GNU General Public License v2.
+# See /LICENSE for more information.
+#
+
+include $(TOPDIR)/rules.mk
+
+PKG_NAME:=cups
+PKG_VERSION:=1.5.4
```

Comment:
This cups version is very old (1.5.4 released July 2012), see https://www.cups.org/blog/2012-07-25-cups-1.5.4.html Please update this to a recent version.